ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/action/AbstractPerspectiveAction.java

```diff
@@ -27,15 +27,19 @@
  */
 public abstract class AbstractPerspectiveAction extends Action {
 
-    private final List<String> activePerspectives;
+    /**
+     * A list of perspectives in which the action is enabled.
+     * Null or empty list means the action is enabled everywhere.
```

Comment:
Disagree with that. AbstractPerspectiveAction was created only to show action in specific perspectives. If you want to create action that will be shown everywhere you can create action that simply extends Action.